lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1040,7 +1040,9 @@ selectCoins ctx gen (ApiT wid) body =
     fmap mkApiCoinSelection
     $ withWorkerCtx ctx wid liftE liftE
     $ \wrk -> do
-        withdrawal <- liftIO $ W.readNextWithdrawal @_ @s @t @k wrk wid
+        -- TODO:
+        -- Allow representing withdrawals as part of external coin selections.
+        let withdrawal = Quantity 0
```

Comment:
I went quite fast in the first iteration here, but this `withdrawal` value actually has no effect because `selectCoinsExternal` can't actually process nor represent them, so changing this to be `0` and left a little TODO.